### PR TITLE
Migrate 08-18

### PR DESCRIPTION
**Pull request by collinhundley**
This PR migrates the package to 08-18 snapshot. 

I'd love for someone to review the changes in `Bind.swift` and `Bind+Node.swift`. The new `UnsafePointer` API is a little unclear to me, though I think I've mostly got it figured it out. The biggest concern is `deinit` in `Bind.swift`, where I had to jump through some hoops to make sure `cBind.buffer` gets cleared correctly.

Input and suggestions are appreciated.

### CODE REVIEW

File: Sources/MySQL/Bind.swift [hunk 1]
```diff
@@ -2,177 +2,221 @@ import Core
...
+    
     /**
-        Creates an output binding from an expected Field.
-    */
+     Creates an output binding from an expected Field.
+     */
     public init(_ field: Field) {
         var cBind = CBind()
-
+
```
Comment (tannernelson) [inline Sources/MySQL/Bind.swift]: What text editor are you using? Seems to be adding spaces to blank lines.

File: Sources/MySQL/Bind.swift [hunk 2]
```diff
@@ -2,177 +2,221 @@ import Core
...
+        
         cBind.buffer_type = field.cField.type
         let length = Int(field.cField.length)
-
+        
         cBind.buffer_length = UInt(length)
-
-        cBind.buffer = UnsafeMutablePointer<Void>.allocate(capacity: length)
+        
+        cBind.buffer = UnsafeMutableRawPointer.allocate(bytes: length, alignedTo: 1)
```
Comment (tannernelson) [inline Sources/MySQL/Bind.swift]: How did you decide `1` for alignment?

File: Sources/MySQL/Bind.swift [hunk 3]
```diff
@@ -2,177 +2,221 @@ import Core
...
+            } else {
+                cBind.buffer.assumingMemoryBound(to: Int64.self).deinitialize()
+            }
+        case MYSQL_TYPE_DOUBLE:
+            cBind.buffer.assumingMemoryBound(to: Double.self).deinitialize()
+        case MYSQL_TYPE_DATE,
+             MYSQL_TYPE_DATETIME,
+             MYSQL_TYPE_TIMESTAMP,
+             MYSQL_TYPE_TIME:
+            cBind.buffer.assumingMemoryBound(to: MYSQL_TIME.self).deinitialize()
```
Comment (tannernelson) [inline Sources/MySQL/Bind.swift]: Do we need to call deinitialize?

File: Tests/MySQL/MySQLTests.swift [hunk 4]
```diff
@@ -1,183 +0,0 @@
-import XCTest
```
Comment (tannernelson) [inline Tests/MySQL/MySQLTests.swift]: Why were the tests deleted?

File: Sources/MySQL/Bind.swift (hunk 1 above)
Comment (collinhundley) [inline Sources/MySQL/Bind.swift]: Just Xcode. That is strange.

**Comment by collinhundley** [comment]
@tannernelson I'll just respond to all your questions here.

For alignment, I'm honestly not sure what should generally be put there. Some information I found online led me to believe that `1` was appropriate for most uses, but I was hoping someone here would have a better idea.

For deinitializing, I'm again not sure. The previous version (07-25) was doing it right before the dealloc, so I'd like to hear why. Unfortunately pointers are probably my weak spot when it comes to Swift.

As for the tests, I have no idea how they got deleted. I'll get that fixed.

**Comment by tannernelson** [comment]
Going to merge this and get tests passing. Thanks!

**Pull request merged**